app/controllers/api/accounts_controller.rb

```diff
@@ -6,23 +6,44 @@
 #  https://github.com/puzzle/cryptopus.
 
 class Api::AccountsController < ApiController
+  helper_method :team
 
+  # GET /api/api_users
   def index
     authorize Account
     accounts = current_user.accounts
     accounts = find_accounts(accounts)
     render_json accounts
   end
 
+  # GET /api/api_users/:id
   def show
-    account = Account.find(params[:id])
     authorize account
     account.decrypt(decrypted_team_password(account.group.team))
     render_json account
   end
 
+  # PATCH /api/api_users/:id?Query
+  def update
+    update_account
+    authorize account
+    account.update!(permitted_attributes(account))
+    account.save
+    render_json account
+  end
+
   private
 
+  def update_account
```

Comment:
I think it's pretty ok just to do this inside #update ... now sep. method required